client/performance-profiler/components/core-web-vitals-display/index.tsx

```diff
@@ -0,0 +1,171 @@
+		bad: translate( "Your site's largest content load needs improvement" ),
+		heading: translate( 'What is largest content load?' ),
+		aka: translate( '(aka Largest Contentful Paint)' ),
+		explanation: translate(
+			'Largest content load measures the time it takes for the largest visible element (like an image or text block) on a page to load. The best sites load in under 2.5 seconds.'
+		),
+	},
+	cls: {
+		good: translate( "Your site's visual stability is good" ),
+		needsImprovement: translate( "Your site's visual stability is moderate" ),
+		bad: translate( "Your site's visual stability needs improvement" ),
+		heading: translate( 'What is visual stability needs?' ),
+		aka: translate( '(aka Content Layout Shift)' ),
+		explanation: translate(
+			'Visual stability is assessed by measuring how often content moves unexpectedly during loading. The best sites have a score of 0.1 or lower.'
+		),
+	},
+	inp: {
+		good: translate( "Your site's interactivity is good" ),
+		needsImprovement: translate( "Your site's interactivity is moderate" ),
+		bad: translate( "Your site's interactivity needs improvement" ),
+		heading: translate( 'What is interactivity?' ),
+		aka: translate( '(aka Interaction to Next Paint)' ),
+		explanation: translate(
+			'Interactivity measures the overall responsiveness of a webpage by evaluating how quickly it reacts to user interactions. A good score is 200 milliseconds or less, indicating that the page responds swiftly to user inputs.'
+		),
+	},
+	ttfb: {
+		good: translate( "Your site's server responsiveness is good" ),
+		needsImprovement: translate( "Your site's server responsiveness is moderate" ),
+		bad: translate( "Your site's server responsiveness needs improvement" ),
+		heading: translate( 'What is server responsiveness?' ),
+		aka: translate( '(aka Time To First Byte)' ),
+		explanation: translate(
+			'Server responsiveness reflects the time taken for a user’s browser to receive the first byte of data from the server after making a request. The best sites load around 800 milliseconds or less.'
+		),
+	},
+};
+
+export const CoreWebVitalsDisplay = ( props: PerformanceMetrics ) => {
+	const translate = useTranslate();
+	const [ activeTab, setActiveTab ] = useState< keyof PerformanceMetrics >( 'lcp' );
+
+	const { displayName } = metricsNames[ activeTab as keyof typeof metricsNames ];
+	const value = props[ activeTab as keyof PerformanceMetrics ];
+	const valuation = mapThresholdsToStatus( activeTab as keyof typeof metricsTresholds, value );
+
+	const { good, needsImprovement } = metricsTresholds[ activeTab as keyof typeof metricsTresholds ];
+	const formatUnit = ( value: number ) => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return +( value / 1000 ).toFixed( 2 );
+		}
+
+		return value;
+	};
+
+	const displayUnit = () => {
+		if ( [ 'lcp', 'fcp', 'ttfb' ].includes( activeTab ) ) {
+			return translate( 'seconds', { comment: 'Used for displaying a range, eg. 1-2 seconds' } );
```

Comment:
Yeah, I think it helps a lot to give context on more general words like those, thanks for adding.